pkg/asset/installconfig/gcp/validation.go

```diff
@@ -133,3 +133,18 @@ func validateMachineNetworksContainIP(fldPath *field.Path, networks []types.Mach
 	}
 	return field.ErrorList{field.Invalid(fldPath, subnetName, fmt.Sprintf("subnet CIDR range start %s is outside of the specified machine networks", ip))}
 }
+
+func CheckForServicesToBeEnabled(projectServices []string) error {
```

Comment:
https://blog.golang.org/godoc

provide good info on how you should document functions are that public API of a package.